src/views/Settings.vue

```diff
@@ -9,7 +9,7 @@
 
     <ion-content>
       <main>
-        <ion-card>
+        <ion-card @click="() => router.push('/profile')">
```

Comment:
routing should be on click of details button on each card.

Reply:
Ok sir, I have removed route from card and added to detail button